projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -717,7 +717,7 @@ public function render_checkbox_multiple_field( $id, $label, $value, $class, $re
 	 */
 	public function render_select_field( $id, $label, $value, $class, $required, $required_field_text ) {
 		$field  = $this->render_label( 'select', $id, $label, $required, $required_field_text );
-		$field .= "\t<select name='" . esc_attr( $id ) . "' id='" . esc_attr( $id ) . "' " . $class . ( $required ? "required aria-required='true'" : '' ) . ">\n";
+		$field .= "\t<select name='" . esc_attr( $id ) . "' id='" . esc_attr( $id ) . "' " . $class . ( $required ? "required aria-required='true'" : '' ) . "aria-hidden='true' tabindex='-1'>\n";
```

Comment:
Looks like there will be a space missing, we'll get `aria-required='true'aria-hidden='true'`. With the way the rest of the string is built, we probably want to do like this
```suggestion
		$field .= "\t<select name='" . esc_attr( $id ) . "' id='" . esc_attr( $id ) . "' " . $class . ( $required ? "required aria-required='true' " : '' ) . "aria-hidden='true' tabindex='-1'>\n";
```

Reply:
Also, thinking about this a bit more... Perhaps we shouldn't put the tabindex and aria-hidden here in the HTML after all, as that will break if someone is using NoScript or the like. Perhaps instead we should add the attributes in [the JS that hides the field](https://github.com/Automattic/jetpack/blob/fix/forms-dropdown-validation/projects/packages/forms/src/contact-form/js/dropdown.js).

Probably the added CSS should also only apply after the JS has run, e.g. by having the JS add a class or make the selector like `.contact-form .contact-form-dropdown[aria-hidden="true"]` or something.

Reply:
✅